tjreports/site/models/reports.php

```diff
@@ -235,11 +235,23 @@ protected function populateState($ordering = '', $direction = 'ASC')
 		$app = JFactory::getApplication();
 		$input = JFactory::getApplication()->input;
 
+		if (!($reportId = $input->get('reportId', 0, 'uint')))
```

Comment:
What is `uint` ?

Reply:
abs of Integer
https://www.php.net/manual/en/function.abs.php